lighthouse-plugin-publisher-ads/audits/script-injected-tags.js

```diff
@@ -92,8 +92,9 @@ class ScriptInjectedTags extends Audit {
       {trace, devtoolsLog}, context);
 
     const closure = getTransitiveClosure(mainDocumentNode, isGptAdRequest);
+    console.log(closure.requests.length);
```

Comment:
Please remove. I think this should have been caught by the presubmit checks.

Reply:
Agreed. Will look into it when I have some time